### PR TITLE
Fix reference to nonexistent package in mdr_listen_action

### PR DESCRIPTION
**Pull request by henrikschnor**
This is a small fix that removes a reference in `setup.py` of `mdr_listen_action` to a python package that no longer exists.

This addresses #117 and #122. The Travis build for this PR should therefore complete successfully.

### CODE REVIEW

File: mdr_planning/mdr_actions/mdr_speech_actions/mdr_listen_action/setup.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
 from catkin_pkg.python_setup import generate_distutils_setup
 
 d = generate_distutils_setup(
-   packages=['mdr_listen_action', 'mdr_listen_action_ros'],
-   package_dir={'mdr_listen_action': 'common/src/mdr_listen_action',
-                'mdr_listen_action_ros': 'ros/src/mdr_listen_action_ros'})
+   packages=['mdr_listen_action_ros'],
+   package_dir={'mdr_listen_action_ros': 'ros/src/mdr_listen_action_ros'})
```
Comment (minhnh) [inline mdr_planning/mdr_actions/mdr_speech_actions/mdr_listen_action/setup.py]: I believe in this case, omitting the package name in the package directory is fine:
```
package_dir={'': 'ros/src/'}
```

Comment (henrikschnor) [reply mdr_planning/mdr_actions/mdr_speech_actions/mdr_listen_action/setup.py]: Good idea, I will try it in the evening.

Comment (argenos) [reply mdr_planning/mdr_actions/mdr_speech_actions/mdr_listen_action/setup.py]: I'll merge with this change as soon as it's ready.

Comment (henrikschnor) [reply mdr_planning/mdr_actions/mdr_speech_actions/mdr_listen_action/setup.py]: Minh's suggestion seems to work, I also added two missing dependencies.

**Comment by minhnh** [comment]
@henrikschnor could you also pull the latest changes from `devel`?

**Pull request merged**